lib/shift/circuit_breaker/circuit_monitor.rb

```diff
@@ -22,10 +22,21 @@ def record_metric(name, state)
         logger.info("* #{metric} *")
       end
 
+      # @param [String] name - The circuit name
+      def record_exception(name)
+        metric = formatted_metric(name)
+        monitor.call(metric) if monitor.respond_to?(:call)
+        logger.info("* #{metric} *")
+      end
```

Comment:
If you intend to keep the new method, then this is pretty much just reimplementing `record_metric`, so we can just reference it instead:

```suggestion
      # @param [String] name - The circuit name
      def record_exception(name)
        record_metric(name, "errored")
      end
```